jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -216,11 +218,11 @@ private Image pullBaseImage(
           .dispatch(
               LogEvent.lifecycle(
                   "The base image reference is manifest list, searching for linux/amd64"));
-      manifestTemplate =
+      manifestAndDigest =
           obtainPlatformSpecificImageManifest(
               registryClient, (V22ManifestListTemplate) manifestTemplate);
+      manifestTemplate = manifestAndDigest.getManifest();
```

Comment:
Nit: this can just be `obtainPlatformSpecificImageManifest(...).getManifest()`. I think we will report the digest of the manifest list in this case. This also avoids reusing the `manifestAndDigest` variable.

Reply:
Yes, we will actually report the digest the switch case following this block under case 2. Since, I was working on both of them together. I initially had the digest log code as well. But later removed it to keep it different in separate PR. Hence, the `manifestAndDigest` variable